classes/PaymentModule.php

```diff
@@ -1104,6 +1104,7 @@ protected function createOrderCartRules(
             // THEN
             //  The voucher is cloned with a new value corresponding to the remainder
             $remainingValue = $cartRule->reduction_amount - $values[$cartRule->reduction_tax ? 'tax_incl' : 'tax_excl'];
+            $remainingValue = Tools::ps_round($remainingValue, Context::getContext()->getComputingPrecision());
```

Comment:
`Context::getContext()->getComputingPrecision()` is already defined on line 1081.
Store it in a variable instead.